wallet/src/ui/app/components/active-coin-card/CoinSelection.module.scss

```diff
@@ -0,0 +1,81 @@
+@use '_utils';
+@use '_variables' as v;
+@use '_values/colors';
+
+.content {
+    display: flex;
+    flex-flow: column nowrap;
+    flex: 1;
+    align-self: stretch;
+    margin: 10px 0;
+
+    .select-coin {
```

Comment:
Since we use modules there is no need to nest selectors, we can use top level selectors and make them simpler and potentially have better performance